comptests/ha/test_restart_pod.py

```diff
@@ -318,8 +542,177 @@ def test_serverpod_status_online_after_replicas(self):
         LOGGER.info("Step 4: Successfully started the pod again by making replicas=1")
         self.restore_pod = False
 
-        #TODO: Step-5 | Getting multiple events for pod delete operation - CORTX-28560
-        #TODO: Step-6 | Pod Online events are not seeing in the health monitor log - CORTX-28867
+        LOGGER.info("Step 5: Check pod online alert and verify node and resource IDs")
+        node_obj = self.ha_comp_obj.get_ha_node_object(self.node_master_list[0])
+        resp_dict = self.ha_comp_obj.get_ha_log_prop(node_obj, common_const.HA_SHUTDOWN_LOGS[2],
+                                                     kvalue=1, health_monitor=True)
+        source_list = resp_dict['source']
+        resource_type_list = resp_dict['resource_type']
+        resource_status_list = resp_dict['resource_status']
+        generation_id_list = resp_dict['generation_id']
+        online_node_id = resp_dict['node_id']
+        online_resource_id = resp_dict['resource_id']
+        for index, (data1, data2, data3) in enumerate(zip(source_list, resource_type_list,
+                                                          resource_status_list)):
+            assert_utils.assert_equal(data1, 'monitor',
+                                      f"Source of {generation_id_list[index]} is not from monitor")
+            assert_utils.assert_equal(data2, 'node',
+                                      f"Resource of {generation_id_list[index]} is not node")
+            assert_utils.assert_equal(data3, 'online',
+                                      f"Resource status of {generation_id_list[index]} "
+                                      "is not online")
+        assert_utils.assert_equal(failed_node_id, online_node_id, "Pod node IDs are differnt")
+        assert_utils.assert_equal(failed_resource_id, online_resource_id, "Pod resource IDs "
+                                                                          "are different")
+        LOGGER.info("Step 5: Successfully checked pod online event to hare and verified the "
+                    "node and resource IDs")
+
+        LOGGER.info("Step 6: Check for publish action event")
+        resp = self.ha_comp_obj.check_string_in_log_file(node_obj, "to component hare",
+                                                         common_const.HA_SHUTDOWN_LOGS[2], lines=4)
+        assert_utils.assert_true(resp[0], "Alert not sent to hare")
+        LOGGER.info("Step 6: Successfully sent action event to hare")
 
         LOGGER.info("COMPLETED: Publish the pod online event to component Hare- "
                     "Data pod comes online after data pod restart using replicas.")
+
+    @pytest.mark.comp_ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-36163")
+    def test_hapod_status_offline_after_replicas(self):
```

Comment:
We are not going to automate these scenario because of some design change. Please remove this.